src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -62,9 +63,9 @@
     private final AtomicBoolean drainRunning = new AtomicBoolean(false);
 
     private LogzioSender(String logzioToken, String logzioType, int drainTimeout, int fsPercentThreshold, String bufferDir,
-                        String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
-                        LogzioLogbackAppender.StatusReporter reporter, ScheduledExecutorService tasksExecutor,
-                        boolean addHostname, String additionalFields, int gcPersistedQueueFilesIntervalSeconds)
+                         String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
```

Comment:
If only spaces added, let's revert to the original indentation please, so we can focus on what's been added